src/methods/modular_programs/hard/q3/UniversityLife.java

```diff
@@ -0,0 +1,37 @@
+package methods.modular_programs.hard.q3;
+
+/*
+Write a program that asks the user for their undergraduate year level and passes it as an integer to a method called
```

Comment:
Can you rework this question text? It is one big run-on sentence currently, and it says that the method prints their status, even though the method actually returns a string.